server/src/api/customers.ts

```diff
@@ -39,4 +40,22 @@ customerRouter.get(
   }
 );
 
+/**
+ * Create a customer if they do not already exist.
+ */
+customerRouter.post(
```

Comment:
May I know why you name this folder as "API"? In my opinion, API sounds like a collection of function definitions waiting to be called by others.
But in this file, customerRouter.get and customerRouter.post are function called that get evaluated once when the file is imported.  They don't look like function definitions; correct me if I get it wrong.

Reply:
@xlai20
> In my opinion, API sounds like a collection of function definitions waiting to be called by others.

This file is exactly what you think API is! :)

> But in this file, customerRouter.get and customerRouter.post are function called that get evaluated once when the file is imported. They don't look like function definitions; correct me if I get it wrong.

^ This is not right. These functions are not being called but these are how the router will handle `post` and `get` requests when it matches the route, which in this case is `/`(as stated in line 47), relative to `/customers/` (because this is a customerRouter, refer [here](https://github.com/googleinterns/gpay-group-buy/blob/master/server/src/index.ts#L36))
Syntax wise, this is just express syntax for routing, you can look at more examples from Express documentation [here](https://expressjs.com/en/4x/api.html#req.baseUrl).

Reply:
I see. Then that's good then.